api/institutions/serializers.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer, RelationshipField, LinksField
+
+class InstitutionSerializer(JSONAPISerializer):
+    name = ser.CharField(required=True)
+    id = ser.CharField(required=True, source='_id')
```

Comment:
Since you're not allowing users to create/modify institutions, I don't know if these need to be required fields.  There's no validation on this data coming in, so it's not being used.